packages/blocks/src/compositionObjects.ts

```diff
@@ -1,4 +1,4 @@
-import { MrkdwnElement, Option, PlainTextElement } from '@slack/types';
+import S from '@slack/types';
```

Comment:
Still a decent amount to do in this file and a few other things to change throughout, but wanted to stop here and break that into a second PR.